src/AWS.Deploy.DockerEngine/DockerEngine.cs

```diff
@@ -197,5 +207,39 @@ public void DetermineDockerExecutionDirectory(Recommendation recommendation)
                 }
             }
         }
+
+        /// <summary>
+        /// Determines the appropriate HTTP port that the underlying container is using.
+        /// </summary>
+        /// <param name="recommendation">The currently selected recommendation</param>
+        /// <returns>The default HTTP port used by the container</returns>
+        public int DetermineDefaultDockerPort(Recommendation recommendation)
+        {
+            switch (recommendation.ProjectDefinition.TargetFramework)
+            {
+                case "net8.0":
+                    return 8080;
+
+                default:
+                    return 80;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether to use a root or non-root user in the underlying container.
+        /// </summary>
+        /// <param name="recommendation">The currently selected recommendation</param>
+        /// <returns>true if a root user is used, else false</returns>
+        private bool UseRootUser(Recommendation recommendation)
+        {
+            switch (recommendation.ProjectDefinition.TargetFramework)
+            {
+                case "net8.0":

```

Comment:
If .NET 8.0 or greater

Reply:
But this means we have to remember to update this code every time there is a new .NET version. Which is easy to forget. I would rather we do a case for "net5.0", "net6.0" and "net7.0" use port 80 and the default to 8080. That way unless Microsoft makes another default port change which would be surprising we don't have to remember to update this code again.